firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -6894,6 +6894,58 @@ extension String {
     }
 }
 
+
+// MARK: - Password generator
+extension String {
+    public struct PasswordGenerator {
+        public static let PasswordGeneratorTitle = MZLocalizedString(
```

Comment:
since we're in the `PasswordGenerator` struct, prefixing the variables with "PasswordGenerator" is probably redundant. What do you think about just "Title"?